### ISSUE TITLE
Accessibility Testing for Public Websites, PACT Act Wizard

### ISSUE THREAD

**Issue by FranECross** [department-of-veterans-affairs/va.gov-team#72031]
### Product information

- [X] Team name, product name, and feature name have been added to the title of this issue.
- [X] Team label, product label and feature label (if applicable) have been added to this issue.

### Who completed the use of color and color contrast test?

@Becapa 

### Use of color and color contrast checks

- [x] All text of 20px or smaller has a 4.5:1 contrast ratio to its background (or better)
- [x] All text of 20px or larger has a 3:1 contrast ratio to its background (or better)
- [x] Non-text elements have a 3:1  contrast ratio to their background and to neighboring elements (or better)
- [x] Color is not the only way to distinguish links from other text (eg. links are underlined)
- [x] Any charts, maps, infographics, and tables convey all information without only relying on color
- [x] Content does not refer to color, especially when providing user instructions (eg. "Click the blue button")

### How did color testing go?

- Normal paragraph and heading text have a contrast of 16.10. 
- Regular links have a contrast of 6.72. Visited links have a contrast of 10.18. 
- Navigation buttons ('Continuee >>', '<< Back') along with selected radio buttons/checkboxes have a contrast of 6.72. 
- Finally, accordion headings have a contrast of 10.18.

### Who completed the axe scans?

@Becapa 

### axe checks

- [x] Each page has been scanned using axe (results shared below)
- [x] axe is integrated into your end-to-end testing

### axe DevTools scan results

0 automatic issues.

### How did axe testing go?

Each page that was scanned showed (0) automatic issues. Axe is being injected and checked in each of the cypress specs for the application (25 specs)

### Who completed the content zoom and reflow test?
@Becapa 

### Content zoom and reflow checks

- [x] All page elements are readable and usable at 200% zoom
- [x] All page elements are readable and usable at 300% zoom
- [x] All page elements are readable and usable at 400% zoom

### How did content zoom and reflow testing go?

Everything was readable by scrolling vertically with each percantage zoom. Nothing fell off the screen horizontally and there was never a need to be able to scroll horizontally.

### Who completed the keyboard navigation test?

@Becapa 

### Keyboard navigation checks

- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element can receive keyboard focus
- [x] Each link, button, form input, checkbox, radio button, select menu, and custom element responds to expected keys
- [x] All elements under focus have a visible focus indicator
- [x] The order of [Tab] stops made sense and was appropriate for completing tasks

### How did keyboard testing go?

Keyboard testing was good. All links were able to be followed by hitting the enter key. I was easily able to Tab and Shift+Tab through form inputs, links, and buttons and the focus always went where it was expected when tabbing.

### Do you have any other results to share?

Advanced testing was completed by our accessibility specialist - @laflannery

**Comment by laflannery** [comment]
Thanks @Becapa! Just a note for @briandeconinck:

For reference, we are purposefully not using DS components here (except `<va-checkbox>`) because `<va-radio>` and <va-checkbox-group>` did not have the flexibility needed in order to have list content as help text. We also couldn't use `<va-radio-option>` in the same way we did the individual checkbox item because the onChange events for the options were dependent on the wrapping component. We do have a [ticket to update the custom HTML](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15910) when the components have been updated and are available for use though.
